<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>策略</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>rps战法</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>海龟战法</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>均线战法</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string> 开始日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_start">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>  结束日期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_end">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_load_data">
           <property name="text">
            <string> 加载数据 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_start">
           <property name="text">
            <string> 开始回测 </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vl_plot_container"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
